odoo/tools/config.py

```diff
@@ -625,22 +615,16 @@ def load(self):
             p.read([self.rcfile])
             for (name,value) in p.items('options'):
                 name = outdated_options_map.get(name, name)
+                option = self.options_index.get(name)
+                if option and not option.file_loadable:
+                    continue
+                if name == 'root_path':
+                    continue
                 if value=='True' or value=='true':
```

Comment:
Cannot do that as it would convert `"0"` and `"1"` to `False` and `True` but that doesn't makes sense for options like `--max-cron-thread`, `--workers` and some others. Also this PR is `[REF]`, adding new allowed values changes the external API and should be `[IMP]`